### DISCUSSION TITLE
Cool CORTX stickerzzz!

### DISCUSSION THREAD

**Discussion by chumakd** [Seagate/cortx#367] [General] [answered]
Wouldn't it be nice if there was a really cool sticker design of _CORTX Community Object Storage_ for the back of a laptop?

Every CORTX community member could proudly show it off to the world!

Just a few nice examples:

![laptop stickers example 1](https://i.pinimg.com/originals/60/6e/9f/606e9f0e3af29ebd789aab96282608fa.jpg)
![laptop stickers example 2](https://i.pinimg.com/originals/aa/ba/d2/aabad26dd758e6d973258cb8fd45352a.jpg)

**Comment by johnbent** [comment, marked as answer]
We have stickers!

Available to anyone who would like one! Request yours by filling out this [form](https://forms.office.com/Pages/ResponsePage.aspx?id=aiFm1EPGSkOcLgV0SMF8vgKFhRIEtppAkBdqKlO8waVUMVk1Uk9PWjc2NVhNTUU1WkdVTTFIVlNMSS4u)
